LandLord-core/src/main/java/biz/princeps/landlord/protection/AOwnedLand.java

```diff
@@ -96,4 +123,14 @@ public boolean contains(Location loc) {
     }
 
     public abstract ILLFlag getFlag(String s);
+
+    protected static String formatNames(Iterable<UUID> uuids) {
+        Set<String> names = new HashSet<>();
+        // ugly, maybe solve this in the future
+        for (UUID uuid : uuids) {
+            names.add(Bukkit.getOfflinePlayer(uuid).getName());
```

Comment:
One idea for this: We could use a StringJoiner

Reply:
I have never heard of it, but sounds nice.